cmd/mattermost/commands/channel.go

```diff
@@ -266,7 +266,7 @@ func removeUserFromChannel(a *app.App, channel *model.Channel, user *model.User,
 }
 
 func removeAllUsersFromChannel(a *app.App, channel *model.Channel) {
-	if err := a.Srv.Store.Channel().PermanentDeleteMembersByChannel(channel.Id); err != nil {
+	if err := a.Srv().Store.Channel().PermanentDeleteMembersByChannel(channel.Id); err != nil {
```

Comment:
If we decide to have the `Store()` getter, I think we should avoid using the `Srv().Store`, but is this kind of confusion the one I'd like to avoid.

I think I prefer directly not having the `Store()` getter shortcut to allow for only one way of doing things